src/xrCore/LocatorAPI.cpp

```diff
@@ -603,6 +632,11 @@ void CLocatorAPI::ProcessOne(pcstr path, const _finddata_t& entry)
 #endif
     xr_fs_strlwr(N);
 
+    bool ignore = ignore_name(N);
+    if (ignore) {
+        return;
+    }
```

Comment:
Excessive variable, parentheses should be on the next line, but in this case the are not even needed.
The code should look like this:
```cpp
if (ignore_name(N))
    return;
```

Reply:
Done